application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -115,6 +150,81 @@ static boolean handleHasBotPermissions(@NotNull String actionVerb,
         return true;
     }
 
+    private static void handleAbsentTarget(@NotNull String actionVerb, @NotNull Interaction event) {
```

Comment:
Imo it should ask for the enum entry instead of the `actionVerb` and get the action verb from the entry

Reply:
this pattern is the same everywhere in this class. i would have to change all methods in this class to follow the same style then. generally, i am okay with this change but i suggest to have it as extra PR instead. (create a GH issue, could even tag it with "good first issue")